WordPress/Classes/Services/LocalCoreDataService.m

```diff
@@ -1,3 +1,4 @@
+#import <CoreData/CoreData.h>
```

Comment:
Would `@import CoreData;` have worked here?

Reply:
I replaced these import with `@import` in all of the changed files from the PR.

Reply:
Thank you for considering my suggestion 🙇‍♂️